app/src/main/java/com/imaginaryrhombus/proctimer/application/TimerRemoteConfigClient.kt

```diff
@@ -0,0 +1,61 @@
+package com.imaginaryrhombus.proctimer.application
+
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.imaginaryrhombus.proctimer.constants.TimerRemoteConfigCliantConstants
+
+/**
+ * Firebase RemoteConfig からの情報取得用クライアントクラス.
+ */
+class TimerRemoteConfigClient: TimerRemoteConfigClientInterface {
+
+    /**
+     * RemoteConfig 本体.
+     */
+    private val remoteConfig =
+        requireNotNull(FirebaseRemoteConfig.getInstance()).apply {
+            setDefaults(HashMap<String, Any>().apply {
+                put(TimerRemoteConfigCliantConstants.versionKey, "0.0.0.0")
+                put(TimerRemoteConfigCliantConstants.storeUrlKey, "")
+                put(TimerRemoteConfigCliantConstants.privacyPolicyKey, "")
+            })
+        }
+
+    /**
+     * RemoteConfig から情報を取得する.
+     * @param cacheExpireSeconds キャッシュが無効化される時間を設定.
+     * @param preApply 情報取得後、適用前に実行される.
+     * @param postApply 情報取得後、適用後に実行される.
+     * @note 情報取得に失敗した場合はどちらも実行されない.
+     */
+    override fun fetchRemoteConfig(cacheExpireSeconds: Long, preApply: () -> Unit, postApply: () -> Unit) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/85662f57ce505e613f53a97ddbeebce37ccafc7f/app/src/main/java/com/imaginaryrhombus/proctimer/application/TimerRemoteConfigClient.kt#L30"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>